### PR TITLE
Appeals structure is flattened

### PR DESCRIPTION
**Pull request by sharonwarner**
Connects #3728 

Part 3/3

- [ ] Confirm all appeals information shows up as expected
- [ ] Confirm we can still delete, add, edit issues for multiple appeals streams

### CODE REVIEW

File: client/app/hearings/components/HearingWorksheetStream.jsx [hunk 1]
```diff
@@ -14,12 +14,10 @@ class HearingWorksheetStream extends Component {
...
-    this.props.worksheetStreams.forEach((appeal) => {
-      appeal.worksheet_issues.forEach((issue) => {
-        if (issue.vacols_sequence_id > maxValue) {
-          maxValue = issue.vacols_sequence_id;
-        }
-      });
+    _.forEach(this.props.worksheetIssues, (issue) => {
+      if (issue.vacols_sequence_id > maxValue) {
+        maxValue = issue.vacols_sequence_id;
+      }
```
Comment (nolawi) [inline client/app/hearings/components/HearingWorksheetStream.jsx]: is this right?
screen shot 2017-10-25 at 5 21 22 pm

File: client/app/hearings/reducers/index.js [hunk 2]
```diff
@@ -56,12 +56,12 @@ export const hearingsReducers = function(state = mapDataToInitialState(), action
     const worksheetIssues = _(worksheetAppeals).flatMap('worksheet_issues').
       keyBy('id').
       value();
-    // TODO: After updating the reducers, we need to remove appeals_ready_for_hearing from the worksheet object
+    const worksheet = _.omit(action.payload.worksheet, ['appeals_ready_for_hearing']);
```
Comment (nolawi) [inline client/app/hearings/reducers/index.js]: cool

**Comment by nolawi** [comment]
I think we might also need the edited flag on newly added issues for `autosave` feature.

File: client/app/hearings/components/HearingWorksheetStream.jsx (hunk 1 above)
Comment (sharonwarner) [reply client/app/hearings/components/HearingWorksheetStream.jsx]: Yikes. That bug has been there for awhile. Fixed now!

**Comment by sharonwarner** [comment]
We should be ok without the edited flag because we're saving immediately. If they edit any of the fields, then the edited flag will be set.

**Pull request merged**